flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -485,6 +485,18 @@ public class KubernetesConfigOptions {
                     .withDescription(
                             "The user agent to be used for contacting with Kubernetes APIServer.");
 
+    public static final ConfigOption<Boolean> KUBERNETES_HADOOP_CONF_MOUNT_DECORATOR_ENABLED =
```

Comment:
Please add a comment above this that these are recommended to be set false when running through the Kubernetes Operator. 
For the Kerberos one I am leaning towards even adding it to the description, I will leave the choice up to you. 😄

Reply:
OK, adding it to the description to make it more clear together w/ config re-generation :)